comptoir-ws/src/main/java/be/valuya/comptoir/ws/rest/control/SaleResource.java

```diff
@@ -141,8 +141,6 @@ public WsSaleRef closeSale(@PathParam("id") long id) {
     @Path("{id}/payed")
     public BigDecimal getSaleTotalPayed(@PathParam("id") long id) {
         Sale sale = saleService.findSaleById(id);
-        saleStateChecker.checkState(sale, false); // TODO: replace with bean validation
-
         return saleService.getSaleTotalPayed(sale);
```

Comment:
Let's remove that then ;-)

Le mer. 26 août 2015 à 13:57, cghislai <notifications@github.com> a écrit :

> In
> comptoir-ws/src/main/java/be/valuya/comptoir/ws/rest/control/SaleResource.java
> <https://github.com/Valuya/comptoir/pull/56#discussion_r37973150>:
>
> > @@ -141,8 +141,6 @@ public WsSaleRef closeSale(@PathParam("id") long id) {
> >      @Path("{id}/payed")
> >      public BigDecimal getSaleTotalPayed(@PathParam("id") long id) {
> >          Sale sale = saleService.findSaleById(id);
> > -        saleStateChecker.checkState(sale, false); // TODO: replace with bean validation
> > -
> >          return saleService.getSaleTotalPayed(sale);
>
> yes, when displaying sales from the history
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Valuya/comptoir/pull/56/files#r37973150>.
>
-- 
Yannick Majoros
Valuya sprl